lib/src/charcodes.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// Generated using:
+//  pub global run charcode \$=dollar \'=single_quote \"=double_quote \
+//      \' '\\\n"$`# \t'
+
+/// "Horizontal Tab" control character, common name.
+const int $tab = 0x09;
+
+/// "Line feed" control character.
+const int $lf = 0x0a;
+
+/// Space character.
+const int $space = 0x20;
+
+/// Character `"`, short name.
+// ignore: constant_identifier_names
+const int $double_quote = 0x22;
```

Comment:
Why not `doubleQuote`? Seems silly to dance around our lint.

Reply:
I didn't try to rename things, just introduce the original names here.
We could rename them now. Let's do that.